sdks/v4-sdk/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@uniswap/v4-sdk",
+  "description": "⚒️ An SDK for building applications on top of Uniswap V4",
+  "repository": "https://github.com/Uniswap/sdks.git",
+  "keywords": [
+    "uniswap",
+    "ethereum"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "module": "dist/v4-sdk.esm.js",
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "tsdx build",
+    "lint": "tsdx lint src",
+    "release": "semantic-release",
+    "start": "tsdx watch",
+    "test": "tsdx test"
+  },
+  "dependencies": {
+    "@ethersproject/abi": "^5.5.0",
+    "@ethersproject/solidity": "^5.0.9",
```

Comment:
nit: wondering if we can upgrade v4-sdk dependencies to ethers 6+.

(on the flip side, it will make merging all the sdks into a single npmjs artifact harder, since all existing sdks rely on ethers 5. So I can be persuaded otherwise)

Reply:
this would make this package bigger, right? Since we'd be importing all of ethers?